core/validation/specific_validation/pathfinders/round_1.py

```diff
@@ -174,6 +175,7 @@ def _check_standard_outputs(
             sheet="Outputs",
             section="Outputs",
             description=PFErrors.STANDARD_OUTPUT_NOT_ALLOWED.format(output=output),
+            cell_index=f"C{breaching_row_indices.pop(0) + 1}",
```

Comment:
This is incremented by 1 due to header row or some such? Should we clarify what this is actually doing somehow? Either in a comment, by unpacking it into a suitable var name prior to the return, or adding to docstring? Otherwise, without any context it's hard to know what is happening here and why

Reply:
It's because DataFrames are 0-indexes and the spreadsheet is not I believe. Can add a comment